--- Canna.conf.dist
+++ Canna.conf
@@ -108,12 +108,12 @@
 # define DefErrDir $(cannaPrefix)/log
 # define DefLibCannaDir $(cannaExecPrefix)/lib
 #else 
-# define DefCannaPrefix /usr/local/canna
+# define DefCannaPrefix /usr
 # ifdef ObsoleteInstallDir
 #  define DefErrDir /usr/spool/canna
 #  define DefLibCannaDir /usr/lib
 # else
-#  define DefErrDir /var/log
+#  define DefErrDir /var/log/canna
 #  define DefLibCannaDir $(cannaExecPrefix)/lib
 # endif
 #endif
@@ -143,11 +143,8 @@
 cannaExecPrefix = $(cannaPrefix)
 cannaBinDir = $(cannaExecPrefix)/bin
 cannaSrvDir = DefCannaSrvDir
-XCOMM cannaLibDir = /var/lib/canna
-XCOMM cannaLibDir = /var/db/canna
-cannaLibDir = DefCannaLibDir
-XCOMM cannaManDir = $(cannaPrefix)/share/man
-cannaManDir = $(cannaPrefix)/man
+cannaLibDir = /var/lib/canna
+cannaManDir = $(cannaPrefix)/share/man
 cannaIncDir = $(cannaPrefix)/include/canna
 
 libCannaDir = DefLibCannaDir
@@ -175,14 +172,10 @@
 #endif
 
 
-cannaOwner = bin
-cannaGroup = bin
+cannaOwner = canna
+cannaGroup = canna
 
-#ifdef InstallAsUser
 cannaOwnerGroup =
-#else
-cannaOwnerGroup = -o $(cannaOwner) -g $(cannaGroup)
-#endif
 
 cannaShLibMajor = 1
 cannaDsoRev = $(cannaShLibMajor).2
@@ -200,15 +193,9 @@
 /* iroha.t の場所を指定。pubdic+ があればそちらを指定します。 */
 pubdicDir = $(CANNAROOT)/dic/ideo/pubdic
 
-#ifdef InstallAsUser
-CHOWN = :
-CHGRP = :
-CHMOD = :
-#else
-CHOWN = chown
-CHGRP = chgrp
+CHOWN = echo
+CHGRP = echo
 CHMOD = chmod
-#endif
 
 /* 日本語マニュアルを使わないのであればコメントアウト */
 #define JAPANESEMAN
