--- cmd/catdic/can.c.str
+++ cmd/catdic/can.c
@@ -1266,7 +1266,7 @@
 	break;
       default:
 	if (init[0] == '/') {
-	    (void)fprintf(stderr,msg_abnl);
+	    (void)fprintf(stderr,"%s",msg_abnl);
 	} else {
 	    (void)fprintf(stderr,msg_abnls,init);
 	}
@@ -1328,11 +1328,11 @@
 
     if ( opt_fq ) {
       if ( opt_s ) {
-        (void) fprintf(stderr,msg_sfq);
+        (void) fprintf(stderr,"%s",msg_sfq);
         exit(ERR_VALUE) ;
       }  
       if ( opt_l || opt_std ) {
-        (void) fprintf(stderr,msg_l);
+        (void) fprintf(stderr,"%s",msg_l);
         exit(ERR_VALUE) ;
       }
       mode = mode | PL_DIC ;
@@ -1524,7 +1524,7 @@
     break;
   default:
     if(init[0] == '/') {
-      fprintf(stderr,msg_abnl );
+      fprintf(stderr,"%s",msg_abnl );
     }
     else {
       fprintf(stderr,msg_abnls,init);
--- cmd/catdic/rutil.c.str
+++ cmd/catdic/rutil.c
@@ -288,7 +288,7 @@
 	 "Dictionary \"%s\" does not exist.\n"), dicname);
     break;
   case NOTALC :
-    (void)fprintf(stderr, msg_mem);
+    (void)fprintf(stderr, "%s", msg_mem);
     break;
   case BADF :
     (void)fprintf(stderr, gettxt("cannacmd:196",
@@ -314,7 +314,7 @@
 	(void)fprintf(stderr,msg_abnls,init);
     }
     else {
-	(void)fprintf(stderr,msg_abnl);
+	(void)fprintf(stderr,"%s", msg_abnl);
     }
     break;
   }
@@ -339,7 +339,7 @@
 	(void)fprintf(stderr, msg_abnls, init);
     }
     else {
-	(void)fprintf(stderr, msg_abnl);
+	(void)fprintf(stderr, "%s", msg_abnl);
     }
     ret = -1;
     return ret;
@@ -350,13 +350,13 @@
 	  (void)fprintf(stderr, msg_mnts, init);
       }
       else {
-	  (void)fprintf(stderr, msg_mnt);
+	  (void)fprintf(stderr, "%s", msg_mnt);
       }
       ret = -1;
       return ret;
     }
     if ((mode & PL_DIC) == PL_DIC ) {
-      (void)fprintf(stderr,msg_fq);
+      (void)fprintf(stderr,"%s", msg_fq);
       ret = -1 ;
       return ret ;
     }
@@ -377,7 +377,7 @@
     ret = 0;
     break;
   case NOTALC :
-    (void)fprintf(stderr, msg_mem);
+    (void)fprintf(stderr, "%s", msg_mem);
     ret = -1;
     break;
   case BADF :
@@ -439,7 +439,7 @@
     ret = -1;
     break;
   case BADCONT :
-    (void)fprintf(stderr,msg_cnt);
+    (void)fprintf(stderr,"%s", msg_cnt);
     ret = -1;
     break;
   default:
@@ -447,7 +447,7 @@
       (void)fprintf(stderr, msg_abnls, init);
     }
     else {
-	(void)fprintf(stderr, msg_abnl);
+	(void)fprintf(stderr, "%s", msg_abnl);
     }
     exit(ERR_VALUE);
     break;
@@ -466,7 +466,7 @@
     ret = 0;
     break;
   case NOTALC :
-    (void)fprintf(stderr, msg_mem);
+    (void)fprintf(stderr, "%s", msg_mem);
     ret = -1;
     break;
   case BADF :
@@ -527,11 +527,11 @@
     ret = -1;
     break;
   case BADARG :
-    (void)fprintf(stderr,msg_fq);
+    (void)fprintf(stderr,"%s", msg_fq);
     ret = -1;
     break;
   case BADCONT :
-    (void)fprintf(stderr,msg_cnt);
+    (void)fprintf(stderr,"%s", msg_cnt);
     ret = -1;
     break;
   case NOENT  :   /* 91.12.03 */
@@ -544,7 +544,7 @@
       (void)fprintf(stderr, msg_abnls, init);
     }
     else {
-	(void)fprintf(stderr, msg_abnl);
+	(void)fprintf(stderr, "%s", msg_abnl);
     }
     exit(ERR_VALUE);
     break;
@@ -574,7 +574,7 @@
       (void)fprintf(stderr, msg_abnls,init);
     }
     else {
-	(void)fprintf(stderr, msg_abnl);
+	(void)fprintf(stderr, "%s", msg_abnl);
     }
     ret = -1;
     return ret;
@@ -585,13 +585,13 @@
 	(void)fprintf(stderr, msg_mnts, init);
       }
       else {
-	  (void)fprintf(stderr, msg_mnt);
+	  (void)fprintf(stderr, "%s", msg_mnt);
       }
       ret = -1;
       return ret;
     }
     if ((mode & PL_DIC) == PL_DIC ) {
-      (void)fprintf(stderr, msg_fq);
+      (void)fprintf(stderr, "%s", msg_fq);
       ret = -1 ;
       return ret ;
     }
@@ -626,7 +626,7 @@
     ret = -2;   /* 93.03.03 */
     break;
   case BADCONT :
-    (void)fprintf(stderr, msg_cnt);
+    (void)fprintf(stderr, "%s", msg_cnt);
     ret = -1;
     break;
   default:
@@ -634,7 +634,7 @@
 	(void)fprintf(stderr, msg_abnls, init);
     }
     else {
-	(void)fprintf(stderr, msg_abnl);
+	(void)fprintf(stderr, "%s", msg_abnl);
     }
     ret = -1;
     break;
@@ -669,7 +669,7 @@
     ret = -2;   /* 93.03.03 */
     break;
   case BADCONT :
-    (void)fprintf(stderr, msg_cnt);
+    (void)fprintf(stderr, "%s", msg_cnt);
     ret = -1;
     break;
   default:
@@ -677,7 +677,7 @@
 	(void)fprintf(stderr, msg_abnls, init);
     }
     else {
-	(void)fprintf(stderr, msg_abnl);
+	(void)fprintf(stderr, "%s", msg_abnl);
     }
     ret = -1;
     break;
